playbooks/collect_activity/collect_activity.yml

```diff
@@ -15,13 +15,13 @@
       copy:
         src: stats.py
         dest: /tmp/collect_activity.py
-      become_user: www-data
+      become: yes
```

Comment:
@haikuginger Re "since this is just a test": it isn't just a test.  The playbook is used for production instances; otherwise I wouldn't be that worried.